src/content/developers/docs/nodes-and-clients/archive-nodes/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Ethereum Archive Node
+description: 
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+An archive node is an instance of an Ethereum client with configuration to build an archive of all historical states. It is a useful tool for certain use cases but might be more tricky to run than a full node. 
+
+## Prerequisites {#prerequisites}
+
+You should understand concept of [Ethereum node](/developers/docs/nodes-and-clients/), [its architecture](/developers/docs/nodes-and-clients/node-architecture/). [sync strategies](https://ethereum.org/en/developers/docs/nodes-and-clients/#sync-modes).   practices of [running](/developers/docs/nodes-and-clients/run-a-node/) and [using them](/developers/docs/apis/json-rpc/).
+
+## What is an archive node
+
+To grasp the importance of an archive node, let's clarify the concept of "state."
+Ethereum can be referred to as *transaction-based state machine*. It consists of accounts and applications executing transactions which are changing their state. The global data with information about each account and contract is stored in trie database called state. This is handled by the execution layer (EL) client and includes:
+* Account balances and nonces 
+* Contract code and storage
+* Consensus-related data, e.g. Staking Deposit Contract
+
+To interact with the network, verify and produce new blocks, Ethereum clients have to keep up with chain tip and therfore the current state. EL client configured as a full node verifies and follows the latest state of the network. A full node temporarily caches only past few states, e.g. for last 128 blocks, for case of a chain reorg. But the recent state is all what client needs to verify incoming transactions and use the network.  
+
+> You can imagine the state as a momentary network snapshot at a given block and the archive as a history replay 
+
+Historical states can be safely pruned because they are not necessary for the network to operate and it would be uselessly redundant for client to keep all out-of-date data. States that existed before recent block with all of their history are effectively thrown away. Only historical blockchain data are kept by a full node but they can be executed to generate a historical state. 
```

Comment:
```suggestion
Historical states can be safely pruned because they are not necessary for the network to operate and it would be uselessly redundant for client to keep all out-of-date data. States that existed before some recent block (e.g. 128 blocks before the head) are effectively thrown away. Full nodes only keep occasional historical snapshots they can use to regenerate older states on request. 
```

Reply:
Thanks for adding this, it's explained nicely but I would keep the point that full nodes keep the blockchain data so it doesn't give impression it's just snapshots. 

"Full nodes only keep historical blockchain data (blocks and transactions) and occasional historical snapshots.."